docs/source/pythonapi/examples/mdgxs-part-i.ipynb

```diff
@@ -962,7 +977,7 @@
        "      <td>1</td>\n",
        "      <td>U235</td>\n",
        "      <td>0.013336</td>\n",
-       "      <td>0.003509</td>\n",
+       "      <td>0.000061</td>\n",
```

Comment:
This is simply my poor understanding of the `str.format()` method, as you can see it was printing the relative error twice instead of the mean.  This is now fixed for all instances in `openmc.mgxs` and `openmc.mdgxs`